Yoshi/Yoshi/YoshiInvocation.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  YoshiInvocation.swift
+//  Pods
+//
+//  Created by Quentin Ribierre on 10/24/16.
+//
+//
+
+import Foundation
+
+/// Set of Yoshi invocation method.
+public struct YoshiInvocation: OptionSetType {
+
+    /// An `OptionSet`'s `Element` type is normally `Self`.
+    public let rawValue: UInt64
+
+    /// Convert from a value of `RawValue`, succeeding unconditionally.
+    public init(rawValue: UInt64) {
+        self.rawValue = rawValue
+    }
+
+    /// All invocations types.
+    public static let all = YoshiInvocation(rawValue: 0)
```

Comment:
Might be better if this was `[. shakeMotionGesture, .multiTouch, .forceTouch]` instead of its own option. You wouldn't need to do a separate check for `.all` in the `shouldShow()` function then.